### PR TITLE
Add make run

### PR DESCRIPTION
**Pull request by cagaser**
:information_source:
Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).

----------------

* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by cagaser** [comment]
Hello@bgruening, I updated  the meta.yaml. I increased the build number and a comment why make was needed.

**Comment by dpryan79** [comment]
I'm not sure I understand the comment regarding `make`. Can you elaborate in a comment here?

**Comment by cagaser** [comment]
Yes @dpryan79 , I am trying to use the clever container (quay.io/biocontainers/clever-toolkit:2.4--py36_boost1.64_0) in a WDL workflow and I get the following message instead: 

```
===== Checking dependencies =====
Found executable /usr/local/bin/ctk-version
Found executable /usr/local/bin/clever-core
Found executable /usr/local/bin/split-priors-by-chromosome
Found executable /usr/local/bin/postprocess-predictions
Could not locate executable "make". It's not in your PATH.
```

**Comment by dpryan79** [comment]
Can you add `make` to `clever-toolkit` instead? It's not obviously needed by delly itself.

**Comment by cagaser** [comment]
@dpryan79 I'm so sorry! I should have double checked.

File: recipes/delly/meta.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ requirements:
     - bzip2
     - libboost
     - htslib
+    - make #Clever container checks at the beginning whether executable make is in PATH.
```
Comment (dpryan79) [inline recipes/delly/meta.yaml]: Please revert the changes to this file then :)

**Comment by dpryan79** [comment]
Thanks :)

**Comment by cagaser** [comment]
Thank you!

**Comment by dpryan79** [comment]
Thanks @cagaser !

**Comment by cagaser** [comment]
Where can I find the updated version of its biocontainer? 
It's not yet updated here: https://quay.io/repository/biocontainers/clever-toolkit?tab=tags

**Comment by dpryan79** [comment]
@bgruening Is this another one of those "the container was uploaded but no one was allowed to see it" permission issue cases?

**Comment by bgruening** [comment]
No, it was also not uploaded to https://anaconda.org/bioconda/clever-toolkit/files

The "the container was uploaded but no one was allowed to see it" case is only per repository once its open it stays open.

**Comment by dpryan79** [comment]
Ah, that explains it. I'll look into why that didn't happen.

**Comment by bgruening** [comment]
https://circleci.com/gh/bioconda/bioconda-recipes/49073 nothing to be done on the master branch.

**Comment by dpryan79** [comment]
Oh, the recipe was just skipped, probably the build number should have been bumped.

**Comment by dpryan79** [comment]
I've merged in an update that should fix this. Let me know if the package doesn't show up in ~30 minutes.

**Comment by cagaser** [comment]
@dpryan79 It's there! Thank you!
